### DISCUSSION TITLE
Include only a specific Package pattern from Docker file

### DISCUSSION THREAD

**Discussion by rajat2911** [renovatebot/renovate#28201] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

Team , I have a below Dockerfile , lets say for example 
```
FROM tomcat:xxxx 
ENV NEWRELIC_VERSION=xxxx

curl -o /usr/local/tomcat/newrelic/newrelic.jar https://download.newrelic.com/newrelic/java-agent/newrelic-agent/${NEWRELIC_VERSION}/newrelic-agent-${NEWRELIC_VERSION}.jar
```

**Comment by secustor** [comment, marked as answer]
Your packageRule doesn't do anything as you do not set any configuration. 

You can deactivate everything else and than enable only for the specific dependencies.

https://docs.renovatebot.com/configuration-options/#enabled

**Reply by rajat2911** [reply]
Thanks for the answer
